spec/core_helpers.rb

```diff
@@ -22,13 +22,9 @@ def core_cache
 
   def populate_core(cache)
```

Comment:
We only want one `populate_core` method in this file. All auth/vocab terms go in the same method, so terms can be reused across all core tests.